src/modules/cpu/kernel/blend.hpp

```diff
@@ -0,0 +1,1103 @@
+#include "rppdefs.h"
+#include "cpu/rpp_cpu_simd.hpp"
+#include "cpu/rpp_cpu_common.hpp"
+
+RppStatus blend_u8_u8_host_tensor(Rpp8u *srcPtr1,
+                                  Rpp8u *srcPtr2,
+                                  RpptDescPtr srcDescPtr,
+                                  Rpp8u *dstPtr,
+                                  RpptDescPtr dstDescPtr,
+                                  Rpp32f *alphaTensor,
+                                  RpptROIPtr roiTensorPtrSrc,
+                                  RpptRoiType roiType,
+                                  RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault;
```

Comment:
it is better to initialize roiDefault in one line = {0, 0, w, h}, by doing so compiler will optimize it better.

Reply:
Done. Changed all instances.